hyperspy/drawing/utils.py

```diff
@@ -600,7 +608,28 @@ def plot_images(images,
         value. It must be in the range [0, 100]
         See :py:func:`numpy.percentile` for more explanation.
         If None, use the percentiles value set in the preferences.
-        If float of integer, keep this value as bounds.
+        If float or integer, keep this value as bounds.
+    overlay : boolean, optional
+        If True, overlays the images with different colors rather than plotting
+        each image as a subplot.
+    colors : list of char or hex string, optional
+	    `colors` should be a list compromising either characters or hex
+		strings, corresponding to colors acceptable to matplotlib. Details
+		can be found at https://matplotlib.org/2.0.2/api/colors_api.html.
+        If 'auto', colors will be taken from matplotlib.colors.BASE_COLORS.
+    alphas : float or list of floats, optional
+        `alphas` should be a single float value or a list of floats 
+        corresponding to the alpha value of each color.
+    legend_picking: bool, optional
+        If True (default), a spectrum can be toggled on and off by clicking on
+        the legended line.
+    legend_loc : {str, int}, optional
+        This parameter controls where the legend is placed on the figure;
+        see the pyplot.legend docstring for valid values
+    pixsize_factor : int, optional
+        Sets the size of the figure when plotting an overlay image. The higher
```

Comment:
A float can also be a valid input and it would be good to specify the default value. Mention that this parameter will be ignored if `fig` if provided.